### ISSUE TITLE
Fishing intensity = 100% forecast tuning

### ISSUE THREAD

**Issue by aaronmberger-nwfsc** [pacific-hake/hake-assessment#1145]
Per the SRG comment, we can try and tune this (lower the threshold) a bit to get it so the in the first year the reported fishing intensity is closer to 100% (currently 103%).  I also suggest we change the label for that forecast to "Fishing intensity near 100%" to cover us if it doesn't exactly = 100% given the threshold search algorithm.

**Comment by cgrandin** [comment]
I started running the forecasting with a lowered threshold (0.0001). So it should be exactly 100% (hope)

**Comment by aaronmberger-nwfsc** [comment]
Given your re-run Chris didn't solve the issue, I figured we would leave this as is for this year.  I went ahead and changed the labe for this forecast row in the D-tables to 'Fishing intensity **near** 100%'.  Calling it good enough for this year.  If the JMC wants  a run where FI = 100% we can do the run for that meeting, though the probability of that occurring is like Pr(JMC ASK RUN) = 0.001.  Thus, closing.
